run/sigterm-handler/main.go

```diff
@@ -27,6 +27,8 @@ import (
 	"time"
 )
 
+var SignalChan chan (os.Signal)
```

Comment:
Since this is in the `main` package, this shouldn't be exported.

Reply:
Also, why not initialize the chan here as well?

Reply:
Fixed, on both counts.